api/handle_create2transfer.go

```diff
@@ -28,20 +28,19 @@ func (a *API) handleCreate2Transfer(create2TransferDTO dto.Create2Transfer) (*co
 		return nil, err
 	}
 	create2Transfer.Hash = *hash
-
 	create2Transfer.SetReceiveTime()
+
+	defer logReceivedTransaction(create2TransferDTO)
```

Comment:
Let's log the hash of received tx as well. We were missing this info when debugging using logs.